rook/run_query/entrypoint.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+set -eu
+
+# export the lib path
+export LD_LIBRARY_PATH=/usr/lib64/ceph
+
+# install required packages
+yum -y update
+yum install -y wget bc
+
+# download the rados-store-glob.sh script
+wget https://raw.githubusercontent.com/uccross/skyhookdm-ceph/skyhook-luminous/src/progly/rados-store-glob.sh > /dev/null 2>&1
+chmod +x ./rados-store-glob.sh
+
+# delete any zombie pool and create a new pool with 128 min and max PG
+ceph osd pool delete tpchdata tpchdata --yes-i-really-really-mean-it
+ceph osd pool create tpchdata 128 128 replicated
```

Comment:
it'd be useful to have `replicated` be exposed as an environment variable so that it can be tweaked at the level of the workflow via the `env` step attribute

Reply:
I see.

Reply:
Done